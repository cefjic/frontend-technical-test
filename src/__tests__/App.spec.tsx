import { render, cleanup } from "@testing-library/react";
import App from "../pages";

const useRouter = jest.spyOn(require("next/router"), "useRouter");

afterAll(cleanup);

describe("App", () => {
  it("should render correctly App", () => {
    useRouter.mockImplementationOnce(() => ({
      asPath: "/",
    }));

    const { container } = render(<App users={[]} conversations={[]} />);
    expect(container).toMatchSnapshot();
  });
});
