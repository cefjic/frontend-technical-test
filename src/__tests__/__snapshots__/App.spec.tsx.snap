// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly App 1`] = `
<div>
  <nav
    class="Navigationstyles__Wrapper-sc-1blsuek-0 fAnmgd"
  >
    <ul
      class="Navigationstyles__List-sc-1blsuek-1 iVsdtD"
    />
  </nav>
</div>
`;
